.devcontainer/devcontainer.json

```
@@ -1,33 +1,11 @@
-// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
-// https://github.com/microsoft/vscode-dev-containers/tree/v0.222.0/containers/ubuntu
 {
```

```suggestion
// For format details, see https://aka.ms/devcontainer.json. For config options, see the
// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
{
```

Let's restore comments for easier access to relevant documentation. I've included the latest links here.